Runtime/Game/Requests/EntitlementRequests.cs.meta

```diff
@@ -1,2 +1,11 @@
 fileFormatVersion: 2
-guid: 930b5026c070b2d4fb183ec255103c0d
\ No newline at end of file
+guid: 930b5026c070b2d4fb183ec255103c0d
+MonoImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  defaultReferences: []
+  executionOrder: 0
+  icon: {instanceID: 0}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
Did you change anything in the Entitlement file?

Reply:
Fixed